### ISSUE TITLE
Issue installing the spinning up (openai-gym) [Solved]

### ISSUE THREAD

**Issue by jonaswagner2826** [BenGravell/spinningup-helper#3]
This is in reference to these steps:
https://github.com/BenGravell/spinningup-helper/blob/main/windows_guide.md#set-up-openai-gym

Obviously swig needs to be installed for pybox2d to be installed, however I ended up with the same issue that this fix was trying to avoid: https://github.com/openai/spinningup/issues/32

I ended up actually reading through the logs and after some guess and check work, I relized that it was trying to use box2d-py as opposed to pybox2d... therefore I used this command instead:
conda install -c conda-forge box2d-py

TLDR: another issue that I probably caused and then fixed...

**Comment by jonaswagner2826** [comment]
are these useful? if so... should I close them each time?

**Comment by BenGravell** [comment]
> This is in reference to these steps:
> https://github.com/BenGravell/spinningup-helper/blob/main/windows_guide.md#set-up-openai-gym
> 
> Obviously swig needs to be installed for pybox2d to be installed, however I ended up with the same issue that this fix was trying to avoid: [openai/spinningup#32](https://github.com/openai/spinningup/issues/32)
> 
> I ended up actually reading through the logs and after some guess and check work, I relized that it was trying to use box2d-py as opposed to pybox2d... therefore I used this command instead:
> conda install -c conda-forge box2d-py
> 
> TLDR: another issue that I probably caused and then fixed...

I tried reproducing this issue. I noticed on my machine running the command 

pip install -e .

from the spinningup directory (which is when gym gets installed) yields this 

```
Obtaining file:///C:/Users/bjgra/Documents/GitHub/spinningup
Collecting gym[atari,box2d,classic_control]~=0.15.3
  Using cached gym-0.15.7-py3-none-any.whl
Collecting pyglet<=1.5.0,>=1.4.0
  Using cached pyglet-1.5.0-py2.py3-none-any.whl (1.0 MB)
Requirement already satisfied: numpy>=1.10.4 in c:\users\bjgra\anaconda3\envs\spintest2\lib\site-packages (from gym[atari,box2d,classic_control]~=0.15.3->spinup==0.2.0) (1.19.5)
Requirement already satisfied: six in c:\users\bjgra\anaconda3\envs\spintest2\lib\site-packages (from gym[atari,box2d,classic_control]~=0.15.3->spinup==0.2.0) (1.15.0)
Requirement already satisfied: cloudpickle~=1.2.0 in c:\users\bjgra\anaconda3\envs\spintest2\lib\site-packages (from gym[atari,box2d,classic_control]~=0.15.3->spinup==0.2.0) (1.2.1)
Requirement already satisfied: scipy in c:\users\bjgra\anaconda3\envs\spintest2\lib\site-packages (from gym[atari,box2d,classic_control]~=0.15.3->spinup==0.2.0) (1.5.3)
Requirement already satisfied: Pillow in c:\users\bjgra\anaconda3\envs\spintest2\lib\site-packages (from gym[atari,box2d,classic_control]~=0.15.3->spinup==0.2.0) (8.2.0)
Collecting atari-py~=0.2.0
  Downloading atari_py-0.2.9-cp36-cp36m-win_amd64.whl (1.6 MB)
     |████████████████████████████████| 1.6 MB ...
Collecting opencv-python
  Using cached opencv_python-4.5.2.52-cp36-cp36m-win_amd64.whl (34.7 MB)
Collecting box2d-py~=2.3.5
  Using cached box2d_py-2.3.8-cp36-cp36m-win_amd64.whl
Collecting future
  Using cached future-0.18.2-py3-none-any.whl
Installing collected packages: future, pyglet, opencv-python, gym, box2d-py, atari-py, spinup
  Running setup.py develop for spinup
Successfully installed atari-py-0.2.9 box2d-py-2.3.8 future-0.18.2 gym-0.15.7 opencv-python-4.5.2.52 pyglet-1.5.0 spinup-0.2.0
```

which indicates that I had some wheels previously built that it used - that might be why I did not suffer the issue with box2d-py.

**Comment by BenGravell** [comment]
> are these useful? if so... should I close them each time?

Yes I think opening & closing issues with your experiences and fixes is a valuable record and immensely useful for ourselves and others who might have the same issues. Keep them coming :)
